server/src/main/java/io/crate/expression/BaseImplementationSymbolVisitor.java

```diff
@@ -92,11 +91,6 @@ public Input<?> visitAlias(AliasSymbol aliasSymbol, C context) {
         return aliasSymbol.symbol().accept(this, context);
     }
 
-    @Override
-    public Input<?> visitVoidReference(VoidReference symbol, C context) {
```

Comment:
Why is this not needed anymore?

Reply:
Sorry my mistake. We still need these for the simple queries like `select o['unknown'] from t;`